### PR TITLE
Interactive run now + simplify docs

### PR DESCRIPTION
**Pull request by benjaoming**
This addition will prompt the user if they wish to run `kalite start` directly after running `kalite setup` in interactive mode.

### CODE REVIEW

File: kalite/distributed/management/commands/setup.py [hunk 1]
```diff
@@ -362,5 +363,11 @@ def handle(self, *args, **options):
             call_command("videoscan")
 
             # done; notify the user.
-            print("CONGRATULATIONS! You've finished setting up the KA Lite server software.")
+            print("\nCONGRATULATIONS! You've finished setting up the KA Lite server software.")
             print("You can now start KA Lite with the following command:\n\n\t%s start\n\n" % start_script_path)
+            
+            if options['interactive']:
+                if raw_input_yn("Do you wish to start the server now?"):
```
Comment (aronasorman) [inline kalite/distributed/management/commands/setup.py]: Doesn't `kalite start` run `setup` already? We also print instructions on how to start the server after this. I feel like adding these lines of code doesn't add much convenience, except for expediting a few more keypresses.

**Comment by aronasorman** [comment]
I'd like to close the 0.14.x now to everything except essential bugfixes. Pushing this to 0.15.

File: kalite/distributed/management/commands/setup.py (hunk 1 above)
Comment (benjaoming) [inline kalite/distributed/management/commands/setup.py]: Yes, but that's in non-interactive mode, so it doesn't affect it.

**Comment by benjaoming** [comment]
@MCGallaspy conflicts resolved, please make note of the docs change.

**Comment by MCGallaspy** [comment]
Noted, rebuilding docs.

**Pull request merged**